airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/config_migrations.py

```diff
@@ -38,7 +38,7 @@ def should_migrate(cls, config: Mapping[str, Any]) -> bool:
             > False, otherwise.
             > Raises the Exception if the structure could not be migrated.
         """
-        return False if config.get(cls.migrate_to_key) else True
+        return False if config.get(cls.migrate_to_key) or not config.get(cls.migrate_from_key) else True
```

Comment:
can we simplify this to
`return cls.migrate_from_key in config and  cls.migrate_from_key not in config`
?

Reply:
i think you meant `return cls.migrate_from_key in config and cls.migrate_to_key not in config`. In your snippet you have `migrate_from_key` twice.

But agree will fix this!